7_kyu/Two_to_One.rb

```diff
@@ -0,0 +1,6 @@
+# https://www.codewars.com/kata/5656b6906de340bd1b0000ac
+
+def longest(a1, a2)
+  (a1 + a2).chars.uniq.sort.join
```

Comment:
Also take a look to `Array#union`: https://ruby-doc.org/core-3.1.0/Array.html#method-i-7C

```ruby
(a1.chars | a2.chars).sort.join
```